src/Schema/PostgreSQLSchemaManager.php

```diff
@@ -632,11 +634,7 @@ protected function selectTableColumns(string $databaseName, ?string $tableName =
                 AND pg_index.indkey[0] = a.attnum
                 AND pg_index.indisprimary = 't'
             ) AS pri,
-            (SELECT pg_get_expr(adbin, adrelid)
-             FROM pg_attrdef
-             WHERE c.oid = pg_attrdef.adrelid
-                AND pg_attrdef.adnum=a.attnum
-            ) AS default,
+            ($fragmentDefaultValueSQL) AS default,
             (SELECT pg_description.description
                 FROM pg_description WHERE pg_description.objoid = c.oid AND a.attnum = pg_description.objsubid
             ) AS comment
```

Comment:
To close the sprintf, I need to do a suggestion on line 652, which is not part of the diff, so that won't work.

Reply:
erf I will do to rebuild a environment to check that (and verify I don't break coding style etc. ) :( 

if the burden is on me can I let you change that in a follow-up PR ? (as I think it's purely a subjective change)

Reply:
As a maintainer, I should be able to push a commit to your branch, hang on.

Reply:
ah yes if you can do so , please do, I don't mind :)